sensors/sensors_darwin_arm64.go

```diff
@@ -96,6 +96,10 @@ func (ta *temperatureArm) getProductNames() []string {
 	ta.ioHIDEventSystemClientSetMatching(uintptr(system), uintptr(ta.sensors))
 	matchingsrvs := ta.ioHIDEventSystemClientCopyServices(uintptr(system))
 
+	if matchingsrvs == nil {
+		return nil
```

Comment:
Returning nil may cause another panic on `dumpNameValues()`. How about return empty value?
```suggestion
		return []string{}
```

Reply:
I know that in Go, an empty slice can be represented as nil. However, due to the subtle differences, I thought that an empty slice was more explicit and therefore preferable. But after reviewing it again, I now see that a nil slice is [considered better](https://go.dev/wiki/CodeReviewComments#declaring-empty-slices), so I retract my previous opinion. Thank you for your comment.